README.md

```diff
@@ -15,17 +15,19 @@ Python wheels are great. Building them across **Mac, Linux, Windows**, on **mult
 What does it do?
 ----------------
 
-|   | macOS 10.9+ x86_64 | manylinux i686 | manylinux x86_64 |  Windows 32bit | Windows 64bit |
+|   | macOS x86_64 | manylinux i686 | manylinux x86_64 | Windows 32bit | Windows 64bit |
 |---|---|---|---|---|---|
-| Python 2.7 | ✅ | ✅ | ✅ | ✅¹ | ✅¹ |
-| Python 3.5 | ✅ | ✅ | ✅ | ✅  | ✅  |
-| Python 3.6 | ✅ | ✅ | ✅ | ✅  | ✅  |
-| Python 3.7 | ✅ | ✅ | ✅ | ✅  | ✅  |
-| Python 3.8 | ✅ | ✅ | ✅ | ✅  | ✅  |
+| Python 2.7      | ✅  | ✅ | ✅ | ✅¹ | ✅¹ |
+| Python 3.5      | ✅  | ✅ | ✅ | ✅  | ✅  |
+| Python 3.6      | ✅  | ✅ | ✅ | ✅  | ✅  |
+| Python 3.7      | ✅  | ✅ | ✅ | ✅  | ✅  |
+| Python 3.8      | ✅  | ✅ | ✅ | ✅  | ✅  |
+| PyPy 2.7 v7.3.0 | ✅² |   | ✅ | ✅  |    |
+| PyPy 3.6 v7.3.0 | ✅² |   | ✅ | ✅  |    |
```

Comment:
missing description for ²

Reply:
Good catch, thanks!

Reply:
But this is not fixed. So PR is not ready..